src/gui/sdlmain.cpp

```diff
@@ -1547,6 +1679,22 @@ static void OutputString(Bitu x,Bitu y,const char * text,Bit32u color,Bit32u col
 //extern void UI_Run(bool);
 void Restart(bool pressed);
 
+/* Get a the postfix of string `string' if its prefix is `prefix' */
+/* Returns: whether `string' has the specified prefix.            */
+static bool GetPostfix( const char* string, const char* prefix, char* postfix )
+{
+	int preflen, stringlen; /* prefix and string lengths */
+
+	preflen   = strlen( prefix  );
+	stringlen = strlen( string );
+
+	if (preflen > stringlen )                     return false;
+	if (strncmp( string, prefix, preflen ) != 0 ) return false;
+
+	strcpy( postfix, string + preflen );
+	return true;
+}
```

Comment:
Remove this function - it's not used any more and generates a warning causing CI to fail. I believe it's the only reason for CI failure right now (but I haven't looked in detail in all logs).

Reply:
Done (not pushed yet).